monix-eval/shared/src/main/scala/monix/eval/Task.scala

```diff
@@ -2675,6 +2676,21 @@ object Task extends TaskInstancesLevel1 {
   def from[F[_], A](fa: F[A])(implicit F: TaskLike[F]): Task[A] =
     F(fa)
 
+  /** Given a `org.reactivestreams.Publisher`, converts it into a
+    * Monix Task.
+    *
+    * See the [[http://www.reactive-streams.org/ Reactive Streams]]
+    * protocol that Monix implements.
+    *
+    * @see [[Task.toReactivePublisher]] for converting a `Task` to
+    *      a reactive publisher.
+    *
+    * @param source is the `org.reactivestreams.Publisher` reference to
+    *        wrap into a [[Task]].
+    */
+  def fromPublisher[A](source: Publisher[A]): Task[Option[A]] =
```

Comment:
We might have this named `fromReactivePublisher` for consistency with `toReactivePublisher` that we already have.
/cc @alexandru